Core1Platform/AdministratingCore1/05_EngineRoom.md

```diff
@@ -0,0 +1,239 @@
+   The vCore utilization is displayed in a curve diagram.
+
+3. Check the limit of vCores assigned to the current instance. To do this, move the mouse pointer on the white line.   
+   The limit of vCores for this instance is displayed in a callout shape.   
+    <!---Hallo Marina, hallo Julian, wisst Ihr, wie die "callout shapes" richtig heißen, bzw. gibt es einen bekannten Namen dafür? ich habe nur diesen Namen gefunden-->   
+ 
+    ![vCore dashlet](../../Assets/Screenshots/Core1Platform/AdministratingCore1/EngineRoomvCores.png "[vCore dashlet]")
+
+4. Check the vCores for any deviations. If desired, move the mouse on a peak of the curve.   
+   The number of vCores is displayed in a callout shape. Used vCores are shown in green, required vCores above the limit for a given time are shown in red.
+
+    ![Number of vCores](../../Assets/Screenshots/Core1Platform/AdministratingCore1/EngineRoomvCoresCurve.png "[Number of vCores]")
+
+
+
+## Check database utilization
+
+Your company has purchased a certain amount of data storage space assigned the main account. If a sandbox is created, it is assigned an amount of storage space from the main account. This reduces the number of storage space available on the main account.   
+
+Actindo has an eye on your data storage utilization. Isolated overruns will have no effect. But if your data storage utilization is permanently over the limit, Actindo will contact you.   
+
+#### Prerequisites
+
+- You have administrator rights for the current instance.
+
+#### Procedure
+
+Actindo Core1 Platform > Any workspace*
+
+![Core1 Platform](../../Assets/Screenshots/Core1Platform/Core1.png "[Core1 Platform]")
+
+
+1. Click the ![Engine room](../../Assets/Icons/EngineRoom.png "[Engine roome]") (Engine room) button center top.  
+   The engine room is opened. The current workspace is overlapped. 
+
+    ![Engine room](../../Assets/Screenshots/Core1Platform/AdministratingCore1/Engineroom.png "[Engine room]")
+
+3. Focus your attention to the dashlet at the second position from the left.  
+   The data storage utilization is displayed in a circle diagram.
+   In the middle of the circle, the current utilization is displayed in percent as well as in gigabyte.
+
+    ![Data storage](../../Assets/Screenshots/Core1Platform/AdministratingCore1/EngineroomDataStorage.png "[Data storage]")
+   >[INFO] Note that these figures are evaluated once a day.   
+
+4. If desired, refresh the current dashlet by clicking the [REFRESH] button to get the current figures.
+
+5. Move your mouse pointer over the circle diagram.   
+   The current data storage utilization is displayed in a callout shape.
+    
+
+
+## Check request statistics
+
+ The request statistics dashlet displays what's going on the current instance. It shows the number of internal requests (jobs) and requests executed from a browser. Additionally, it shows the number of errors occurred during the request processing.   
+ The statistic is evaluated per each hour.
```

Comment:
Warum ist dieser Satz in einem separaten Abschitt?

Reply:
weil es eine andere info ist